.github/workflows/dist.yml

```diff
@@ -126,13 +132,11 @@ jobs:
           fetch-depth: 1
           submodules: recursive
 
-      # NB: CIBW builds wheels in containers on Linux
-      - name: Setup ccache (macOS only)
-        if: runner.os == 'macOS'
+      - name: Setup ccache
         uses: hendrikmuhs/ccache-action@v1
         with:
           create-symlink: true
-          key: ccache-${{ runner.os }}-${{ runner.arch }}-${{ matrix.python-version }}-${{ matrix.target.toolkit }}
+          key: ccache-${{ runner.os }}-${{ runner.arch }}
```

Comment:
Please revert this. See https://github.com/tile-ai/tilelang/pull/1184/files#r2488892874

Reply:
This should be fine. `${{ runner.os }}-${{ runner.arch }}` resolves to `Linux-X86` or `Linux-ARM64`. While most part of tvm is independent to python/CUDA, most part of the cache should be reusable.

Reply:
Let me clarify, for example, when we support more archs, such as `s390x` / `riscv64` with QEMU, the build cache is not usable between archs. While the host Linux runner is still X64, that will give the same cache key prefix. An invalid cache will surpass the valid cache due to a newer timestamp.

------

> those builds can benefits from previous cache

As I mentioned earlier in https://github.com/tile-ai/tilelang/pull/1184#discussion_r2488938500:

> Since we do not change `matrix.python-version` and `matrix.target.toolkit` very often, it will not be a performance problem with a detailed cache key.

Reply:
> for example, when we support more archs, such as `s390x` / `riscv64` with QEMU, the build cache is not usable between archs

While the current `cibuildwheel` config supports cross compiling with qemu, I don't think our GA workflows will involve that.

Even if we want to cover that case, `matrix.python-version` and `matrix.target.toolkit` won't help.